.idea/circuitpython_jupyter_kernel.iml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you please delete the `.idea/` folder, you might've accidentally comitted it?

Reply:
Thanks for the revision, you are right, I might committed my Pycharm folder by accident. I will correct that.